appserver/web/web-glue/src/main/java/com/sun/enterprise/web/WebModuleListener.java

```diff
@@ -265,6 +272,19 @@ private void configureJsp(WebModule webModule) {
 
     }
 
+    private static void initializeChildServices(ServiceLocator serviceLocator, ClassLoader childClassLoader) {
+        DynamicConfigurationService dcs =
+                    serviceLocator.getService(DynamicConfigurationService.class);
+            Populator populator = dcs.getPopulator();
+        try {
+            populator.populate(new ClasspathDescriptorFileFinder(childClassLoader));
+        } catch (IOException ex) {
+            Logger.getLogger(WebModuleListener.class.getName()).log(Level.SEVERE, null, ex);
+        } catch (MultiException ex) {
```

Comment:
Two `catch` blocks log same message. Replace with single `catch` and `|`?

Reply:
Thanks for spotting this. I refactored this a bit now.